trade-finance-manager-ui/server/controllers/case/underwriting/amendments/amendment-underwriter-managers-decision/mapDecisionObject.js

```diff
@@ -0,0 +1,50 @@
+const { hasValue } = require('../../../../../helpers/string');
+const { userFullName } = require('../../../../../helpers/user');
+const CONSTANTS = require('../../../../../constants');
+
+const mapDecisionValue = (decision) => {
```

Comment:
I don't think we need this mapping. previously, for those decisions we had different values in portal and tfm. 
but now we don't have portal anymore. so the decision text should be consistent

Reply:
discussed - not yet in use